irohad/network/ordering_service_transport.hpp

```diff
@@ -17,8 +17,9 @@
 #ifndef IROHA_ORDERING_SERVICE_TRANSPORT_H
 #define IROHA_ORDERING_SERVICE_TRANSPORT_H
 
-#include "model/proposal.hpp"
-#include "model/transaction.hpp"
+#include "backend/protobuf/proposal.hpp"
+#include "backend/protobuf/transaction.hpp"
+#include "interfaces/transaction.hpp"
```

Comment:
No need to include the whole file, you can use forward declaration

Reply:
There's no particular descision so far, [google prefer](https://google.github.io/styleguide/cppguide.html#Forward_Declarations) its avoiding